package.json

```diff
@@ -39,16 +39,16 @@
     "@graphql-codegen/typescript-operations": "^4.0.1",
     "@graphql-codegen/typescript-urql": "^3.7.3",
     "@types/formidable": "^3.4.2",
-    "@typescript-eslint/eslint-plugin": "^6.5.0",
-    "eslint": "8.47.0",
+    "@typescript-eslint/eslint-plugin": "^6.6.0",
+    "eslint": "8.48.0",
     "eslint-config-next": "13.4.19",
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-simple-import-sort": "^10.0.0",
     "next-transpile-modules": "^10.0.1",
     "prettier": "^3.0.2",
-    "strip-ansi": "6.0.1",
+    "strip-ansi": "7.1.0",
```

Comment:
@dan13ram it's just a weird workaround... because `strip-ansi-cjs` is installed from a separate lib, `strip-ansi` for some reason won't automatically get installed for one of graphql type generation libs, which needs `strip-ansi`